x-pack/plugins/security_solution/public/common/containers/events/last_event_time/index.ts

```diff
@@ -96,15 +98,13 @@ export const useTimelineLastEventTime = ({
                 }));
               } else if (isErrorResponse(response)) {
                 setLoading(false);
-                // TODO: Make response error status clearer
-                notifications.toasts.addWarning(i18n.ERROR_LAST_EVENT_TIME);
+                addWarning(i18n.ERROR_LAST_EVENT_TIME);
               }
             },
             error: (msg) => {
```

Comment:
We're still not able to type this are we?

Reply:
If I understand what you're saying correct, we are not able to ingest a partial error/response at this moment which is correct.

```ts
addWarning(i18n.ERROR_LAST_EVENT_TIME);
```

That warning isn't going to be great. However, I also haven't seen that much if any at all and couldn't trip it during testing. I really don't know if/when it gets tripped to be totally honest. I think if we see that happening or we dig into it, we should make a follow up PR where we can add the response into the warning object and then push it onto the toaster if possible:

```ts
addWarning(response, i18n.ERROR_LAST_EVENT_TIME);
```

But won't be soon :-) (at least from myself)